tensorboard/webapp/notification_center/_views/notification_center_component.scss

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+.messages {
+  top: 30px;
+  position: absolute;
+  border: 1px solid gray;
+  right: -100px;
```

Comment:
Why do you have to use `absolute` and such negative `right` value? What does the value signify? Implement using `mat-menu` (https://material.angular.io/components/menu/overview)?

Reply:
Thanks for pointing mat-menu!
This was for quick prototyping. I've empty both ng.html and scss file. Save those for the next pr.